src/main/scala/uk/gov/homeoffice/drt/service/FlightScheduledService.scala

```diff
@@ -23,35 +24,37 @@ class FlightScheduledService[F[_] : Sync](arrivalsRepository: ArrivalRepositoryI
   def transformArrivals(requestedDetails: FlightsRequest, arrivalsTableData: F[List[ArrivalTableDataIndex]]): F[List[Arrival]] = {
```

Comment:
This is another function name that doesn't say what it does. How is it transforming arrivals? Is it taking rows from the arrivals table and turning them into `Arrival`s?

Reply:
I can rename as `transformArrivalsFromArrivalTable` which make is it more obvious .